src/control/functor.lean

```diff
@@ -2,12 +2,28 @@
 Copyright (c) 2017 Simon Hudon. All rights reserved.
 Released under Apache 2.0 license as described in the file LICENSE.
 Author: Simon Hudon
-
-Standard identity and composition functors
 -/
 import tactic.ext
 import tactic.lint
 
+/-!
+# Functors
+
+This module provides additional functor lemmas, definitions, and instances.
+
+## Main definitions
+
+* `const α` is the functor that sends all types to `α`.
+* `add_const` is the functor that sends all types to their `const` functor.
+* `comp F G` for functors `F` and `G` is the functor that is the composition of the functors.
+* `liftp` and `liftr` lift properties and relations, respectively, to a term `x : F α`
```

Comment:
```suggestion
* `liftp` and `liftr` lift properties and relations on a type `α`, respectively, 
  to properties and relations on `F α`, interpreted as something containing 
  elements of type `α`
```

Reply:
@kmill I don't think I put the suggestion in github properly - there was a little more I suggested to change. In particular I think it's not very clear to say a relation is lifted to a "term `x : F α`", so I think it's clearer to say relations on `F α`

Reply:
@b-mehta Good point -- I think I've clarified this now